test/test_helper.rb

```diff
@@ -7,5 +7,6 @@
 require 'pry'
 require 'byebug'
 require 'pry-byebug'
+require 'timecop'
```

Comment:
I would rather get rid of the `timecop` dependency and be able to pass it in an as an optional argument, so that you can mock it. See e.g. how `#eta` is tested.